broadcast/PSO-00-Create-Proxies.sol/42220/run-latest.json

```diff
@@ -0,0 +1,117 @@
+{
+  "transactions": [
+    {
+      "hash": "0x7d0088e4798fee338276bace188c9b44fbb290c6f6ca437e95c01d8d76b977a3",
+      "transactionType": "CREATE",
+      "contractName": "StableTokenPSOProxy",
+      "contractAddress": "0x7242a5FfbC765F322ee7cAd7957c291aC54c33F4",
```

Comment:
this is unverified on celoscan, not sure when/where we usually verify contracts but intuitively I would have expected verification to be part of this PR

Reply:
on it, will merge after 👍